lib/ZumoOta/src/Upload.cpp

```diff
@@ -98,14 +101,36 @@ void Upload::handleFileUpload(AsyncWebServerRequest *request, const String& file
     if (final)
     {
         request->_tempFile.close();
-        /*Check if the file exists in the file system*/
-        if(LittleFS.exists(updatedFilename))
+       /* Check if the file exits in FileSystem */
+        if (LittleFS.exists(updatedFilename))
         {
-            LOG_DEBUG(String(updatedFilename) + " " + "exists in FileSystem.");
+            /* Open the file in read mode */
+            File file = LittleFS.open(updatedFilename, "r");

```

Comment:
Shorter:
file = LittleFS.open(filename, "r");
size_t filesize = file.size();
file.close();

Reply:
It will be performed by the next commit.